pgptool-gui/installer/launch4j.xml

```diff
@@ -2,8 +2,8 @@
 <launch4jConfig>
   <dontWrapJar>true</dontWrapJar>
   <headerType>gui</headerType>
-  <jar>pgptoolgui.jar</jar>
-  <outfile>/home/sergeyk/_prj/pgptool/pgptool-gui/target/pgptool.exe</outfile>
```

Comment:
This PR is now ready for review as I figured out what you were doing wrong with the Windows cross compile. Obviously I have no `/home/sergeyk` on my system :-)